### PR TITLE
feat(1.5): kong, keycloak & multitenancy

### PR DESCRIPTION
**Pull request by obdulia-losantos**

### CODE REVIEW

File: api-auth/service/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  // or are redirected to keycloak to authenticate
+  "oidc_endpoints": [
+    {
+      // endpoint name (unique among rest of OIDC endpoints in this service)
+      "name": "protected",
+
+      // internal url
+      "url": "/protect-me-please",
+
+      // [optional] external url, defaults to "/{realm}/{name}{url}" where
```
Comment (adamvert) [inline api-auth/service/README.md]: This is a little confusing: I guess that `url` refers to the previous setting, which you refer to in the comment as "internal url"? but that `name` doesn't refer to the "endpoint name" above? or am I totally wrong?

File: api-auth/service/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+      // internal url
+      "url": "/i-am-public/",
+
+      // [optional] external url, defaults to "/{realm}/{name}{url}" where
+      // {realm} is the realm name,
+      // {name} is the service name and
+      // {url} is the endpoint url.
+      // use case: if the endpoint does not depend on any realm
+      "route_path": "/my-service/public/",
```
Comment (adamvert) [inline api-auth/service/README.md]: Same again - so would the default here be `/my-service/public/i-am-public/`?

File: docker-compose-base.yml [hunk 3]
```diff
@@ -25,45 +25,16 @@ services:
...
-
+      - 9000:9000
 
   # ---------------------------------
   # Aether kernel
   # ---------------------------------
 
   kernel-base:
-    image: ehealthafrica/aether-kernel:${AETHER_VERSION}
+    image: ehealthafrica/aether-kernel:${AETHER_VERSION:-alpha}
```
Comment (adamvert) [inline docker-compose-base.yml]: We should make sure we change this :)

File: api-auth/service/README.md (hunk 1 above)
Comment (shawnsarwar) [reply api-auth/service/README.md]: These conventions are from Gateway Manager. We should address them over there on a breaking change and then propagate them back: https://github.com/eHealthAfrica/gateway-manager

**Pull request merged**